libs/core/src/scss/components/_grid.scss

```diff
@@ -0,0 +1,97 @@
+@use 'sass:map';
+
+/* stylelint-disable custom-property-pattern */
+$breakpoint-map: (
+  'xs': 0,
+  'sm': 632px,
+  'md': 768px,
+  'lg': 992px,
+  'xl': 1200px,
+);
+
+.kirby-grid {
+  --kirby-grid-columns: 12;
+  --kirby-grid-columnSpacing: 24px;
+  --kirby-grid-rowSpacing: 24px;
+
+  min-width: 0;
+  box-sizing: border-box;
+  display: flex;
+  flex-flow: row wrap;
+  margin: calc(var(--kirby-grid-rowSpacing) / -2) calc(var(--kirby-grid-columnSpacing) / -2);
```

Comment:
Clever! The way I understand this, we use negative margins to be able to add column and row padding consistently around every element even when it is adjacent to the edge of the grid? 

Could we add a comment explaining that, for future us?